pkgs/by-name/op/openjph/package.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "openjph";
+  version = "0.14.2";
+
+  src = fetchFromGitHub {
+    owner = "aous72";
+    repo = pname;
```

```suggestion
    repo = "openjph";
```
we shouldn't conflate the package name with repository name. see https://github.com/NixOS/nixpkgs/issues/277994